sklearn/ensemble/_hist_gradient_boosting/tests/test_gradient_boosting.py

```diff
@@ -249,6 +251,58 @@ def test_absolute_error_sample_weight():
     gbdt.fit(X, y, sample_weight=sample_weight)
 
 
+@pytest.mark.parametrize("y", [([1.0, -2.0, 0.0]), ([0.0, 1.0, 2.0])])
+def test_gamma_y_positive(y):
+    # Test that ValueError is raised if any y_i <= 0.
+    err_msg = r"loss='gamma' requires positive y."
```

Comment:
I know that in English-maths-language positive means "strictly positive" (as opposed to non-negative), but wouldn't it be helpful to non-native English-maths-language speakers to put be extra explicit and use "strictly" here?